src/commands/GameCommand.ts

```diff
@@ -1,17 +1,51 @@
 import { ICommand } from "./ICommand";
+import { BusColor } from "../model/BusColor";
 
 export class GameCommand implements ICommand{
+
     type:string
     data:Object
     privateData:Object
     player:string
     id:number
+    message:string
     constructor(type:string, data:Object, privateData:Object, player:string, id:number){
         this.type=type
         this.data=data
         this.privateData=privateData
         this.player=player
         this.id =id
+        this.message=this.generateMessage();
+    }
+    generateMessage(){
+        if(this.type==="drawBus"){
+            return this.drawBus();
+        } else if(this.type ==="discardRoutes"){
+            return this.drawRoutes();
+        } else if(this.type ==="incrementTurn"){
+            return this.incrementTurn();
+        } else{
+            return "unknown command?!? "+this.player +" "+this.type+" "+this.data;
```

Comment:
I'm not sure what's best to do here, but it may be safer to just make it undefined or a string we can check against on the frontend so that we can easily filter out all of the commands that don't belong in History.